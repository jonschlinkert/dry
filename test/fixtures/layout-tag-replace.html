{% layout "layout-replace.html" %}

This is root-text-node content from "fixtures/layout-replace.html".

{% block "head" %}
head: "fixtures/layout-replace.html"
{% endblock %}

{% block "footer" %}
footer: "fixtures/layout-replace.html"
{% endblock %}
