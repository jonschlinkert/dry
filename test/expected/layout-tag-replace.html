<!DOCTYPE html>
<html lang="en">
  
head: "fixtures/layout-replace.html"
  <body>
    
This is root-text-node content from "fixtures/layout-replace.html".


    
footer: "fixtures/layout-replace.html"
  </body>
</html>